<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionDialog</class>
 <widget class="QWidget" name="TransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="informativeText">
     <property name="text">
      <string/>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextBrowser" name="detailedText">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="text">
     <property name="text">
      <string/>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionRunInTerminal">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/images/terminal.png</normaloff>:/resources/images/terminal.png</iconset>
   </property>
   <property name="text">
    <string>Run in terminal</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
